### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**Please provide a description of this PR:**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -41,6 +41,7 @@ Makefile*                                                        @istio/wg-test-
 /pkg/tracing/                                                    @istio/wg-policies-and-telemetry-maintainers
 /pkg/wasm/                                                       @istio/wg-policies-and-telemetry-maintainers
 /pkg/webhooks/                                                   @istio/wg-environments-maintainers
+/pkg/workloadapi/                                                @istio/wg-networking-maintainers
```
Comment (ericvn) [inline CODEOWNERS:44]: Should workloadapi/security be owned by @istio/wg-security-maintainers?

**Comment by ericvn** [comment]
I assume this is because https://github.com/istio/istio/pull/46945 has T&R as owners.

File: CODEOWNERS (hunk 1 above)
Comment (linsun) [reply CODEOWNERS:44]: seems reasonable to me

Comment (keithmattix) [reply CODEOWNERS:44]: Maybe co-ownership? Specifically because the changes to that directory need to be replicated in ztunnel

**Review by howardjohn — APPROVED** [review]

**Pull request merged**